rust/actyx/ax/README.md

```diff
@@ -0,0 +1,44 @@
+[![Latest Version](https://img.shields.io/crates/v/ax.svg)](https://crates.io/crates/ax)
+[![Rust Documentation](https://docs.rs/ax/badge.svg)](https://docs.rs/ax)
+
+# <img src="https://developer.actyx.com/img/logo.svg" height="32px"> AX Databank and CLI
+
+[AX](https://crates.io/crates/ax) is a decentralized event database, streaming and processing
+engine that allows you to easily build [local-first cooperative](https://www.local-first-cooperation.org/)
+apps.
+It makes it easy to run distributed applications on multiple nodes.
+It is a piece of software that allows you to run your own apps on one or more edge devices and have these apps seamlessly communicate and share data with each other.
+
+---
+
+> **Note:** _AX has been created by the company [Actyx AG](https://developer.actyx.com/) and predates the unrelated [Actix framework](https://crates.io/crates/actix) (although AX wasn’t open-sourced until Oct’23).
+> While we have changed crate names to avoid confusion, the company name remains Actyx AG and appears in some places._
+
+---
+
+## Installation
+
+This is a binary crate, its intended use is via
+
+    cargo install ax
```

Comment:
Not a fan of this style of code blocks, especially given that most (if not all) code blocks we have use triple backticks